### ISSUE TITLE
CHANGE COLUMN is not replicated in ClickHouse when we have multiple databases

### ISSUE THREAD

**Issue by Selfeer** [Altinity/clickhouse-sink-connector#572]
When trying to do MySQL to ClickHouse replication when we monitor all MySQL databases via removing `database.include.list` configuration and trying to perform `ALTER TABLE ... CHANGE COLUMN` the changes do not seem to be replicated in ClickHouse.

We have 5 databases:
`test`, `database_1`, `database_2`, `database_3`, `database_4`

On `database_1`  we create two tables. On the first table we perform `ALTER TABLE ... ADD COLUMN` - this action is successful and is replicated in ClickHouse correctly.
```sql
CREATE TABLE IF NOT EXISTS database_1.`table_fae58e70_0c60_11ef_a293_cfcc8bfdbf55` (id INT NOT NULL,col1 varchar(255), col2 int, PRIMARY KEY (id)) ENGINE = InnoDB;
INSERT INTO database_1.`table_fae58e70_0c60_11ef_a293_cfcc8bfdbf55` VALUES (-1144554638,'SampleText',67329652);
ALTER TABLE database_1.`table_fae58e70_0c60_11ef_a293_cfcc8bfdbf55` ADD COLUMN new_col varchar(255);
```
Now, again, on `database_1` we create another table and perform `ALTER TABLE ... CHANGE COLUMN`.

```sql
CREATE TABLE IF NOT EXISTS database_1.`table_fcdd63fd_0c60_11ef_a293_cfcc8bfdbf55` (id INT NOT NULL,col1 varchar(255), col2 int, PRIMARY KEY (id)) ENGINE = InnoDB;
INSERT INTO database_1.`table_fcdd63fd_0c60_11ef_a293_cfcc8bfdbf55` VALUES (2099428746,'SampleText',-1825730071);
ALTER TABLE database_1.`table_fcdd63fd_0c60_11ef_a293_cfcc8bfdbf55` CHANGE COLUMN col1 new_col varchar(255);
```

The changes are not applied in ClickHouse now:
```sql
SELECT name FROM system.columns WHERE table = 'table_fcdd63fd_0c60_11ef_a293_cfcc8bfdbf55' AND name = 'new_col' AND database = 'database_1' FORMAT TabSeparated
```

In the sink logs we see that in ClickHouse `MODIFY COLUMN` and `RENAME COLUMN` were initiated but it seems that `RENAME COLUMN` syntax is incorrect:

```
2024-05-07 11:00:19.740 INFO  - Executed Source DB DDL: ALTER TABLE database_1.`table_fcdd63fd_0c60_11ef_a293_cfcc8bfdbf55` CHANGE COLUMN col1 new_col varchar(255) Snapshot:false
2024-05-07 11:00:19.740 INFO  - ClickHouse DDL: ALTER TABLE database_1.`table_fcdd63fd_0c60_11ef_a293_cfcc8bfdbf55` MODIFY COLUMN col1 String
2024-05-07 11:00:19.751 INFO  - ClickHouse DDL: ALTER TABLE database_1.`table_fcdd63fd_0c60_11ef_a293_cfcc8bfdbf55` RENAME COLUMN col1 String
``` 

And we get this error in logs:
```
2024-05-07 11:00:19.752 ERROR - Error running DDL Query: java.sql.SQLException: Code: 62. DB::Exception: Syntax error: failed at position 88 ('String'): String. Expected one of: token, Dot, TO. (SYNTAX_ERROR) (version 23.3.22.3 (official build))
```

**Her is the build and config values used:**

**Build:** `altinityinfra/clickhouse-sink-connector:548-4970a5e1f66465876196c0449b913f584724b261-lt`
```yaml
name: my_connector
database.hostname: mysql-master
database.port: '3306'
database.user: root
database.password: root
database.server.name: ER54
clickhouse.server.url: clickhouse
clickhouse.server.user: root
clickhouse.server.password: root
clickhouse.server.port: '8123'
clickhouse.server.database: test
database.allowPublicKeyRetrieval: 'true'
snapshot.mode: initial
offset.flush.interval.ms: '5000'
connector.class: io.debezium.connector.mysql.MySqlConnector
offset.storage: io.debezium.storage.jdbc.offset.JdbcOffsetBackingStore
offset.storage.jdbc.offset.table.name: altinity_sink_connector.replica_source_info
offset.storage.jdbc.url: jdbc:clickhouse://clickhouse:8123/altinity_sink_connector
offset.storage.jdbc.user: root
offset.storage.jdbc.password: root
offset.storage.jdbc.offset.table.ddl: |-
  CREATE TABLE if not exists %s
  (
      `id` String,
      `offset_key` String,
      `offset_val` String,
      `record_insert_ts` DateTime,
      `record_insert_seq` UInt64,
      `_version` UInt64 MATERIALIZED toUnixTimestamp64Nano(now64(9))
  )
  ENGINE = ReplacingMergeTree(_version)
  ORDER BY id
  SETTINGS index_granularity = 8198
offset.storage.jdbc.offset.table.delete: delete from %s where 1=1
schema.history.internal: io.debezium.storage.jdbc.history.JdbcSchemaHistory
schema.history.internal.jdbc.url: jdbc:clickhouse://clickhouse:8123/altinity_sink_connector
schema.history.internal.jdbc.user: root
schema.history.internal.jdbc.password: root
schema.history.internal.jdbc.schema.history.table.ddl: |-
  CREATE TABLE if not exists %s
  (`id` VARCHAR(36) NOT NULL, `history_data` VARCHAR(65000), `history_data_seq` INTEGER, `record_insert_ts` TIMESTAMP NOT NULL, `record_insert_seq` INTEGER NOT NULL) ENGINE=ReplacingMergeTree(record_insert_seq) order by id
schema.history.internal.jdbc.schema.history.table.name: altinity_sink_connector.replicate_schema_history
replacingmergetree.delete.column: _sign
enable.snapshot.ddl: 'true'
database.connectionTimeZone: UTC
database.serverTimezone: UTC
clickhouse.datetime.timezone: UTC
auto.create.tables: 'true'

```

**Comment by subkanthi** [comment]
@Selfeer fixed and merged to `2.1.0`

**Comment by Selfeer** [comment]
The issue is fixed, thank you @subkanthi
